rule_manager.py

```diff
@@ -0,0 +1,135 @@
+from typing import Any, Dict, Optional
+
+from pypanther import LogType, Rule, get_panther_rules, get_rules, register
+from pypanther.wrap import exclude, include
+
+
+class RuleManager:
```

Comment:
It surprises me that this would be defined here, rather than in pypanther/pypanther-core. Can you elaborate on the reasoning?